### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tstirrat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +24,28 @@ You will also see any lint errors in the console.
...
+To run the emulator suite in your project folder:
+
+```bash
+cd project/
+firebase emulators:start --import my-data
+
+✔  hub: emulator hub started at http://localhost:4400
+```
+
+This will run the emulators for you project.<br />
```
Comment (yuchenshi) [inline README.md:38]:
```suggestion
This will run the emulators for your project.<br />
```

File: README.md [hunk 2]
```diff
@@ -26,6 +24,28 @@ You will also see any lint errors in the console.
...
+
+Run the Emulator Suite UI from the firebase-tools-ui folder:
```
Comment (yuchenshi) [inline README.md:40]:
```suggestion
In another terminal, run the Emulator Suite UI from the firebase-tools-ui folder: (note that `<project-id>` must be replaced with the matching project id of your project.)
```

File: README.md [hunk 3]
```diff
@@ -26,6 +24,28 @@ You will also see any lint errors in the console.
 
 NOTE: The `emulators:exec` command is necessary to set the environment variables for the web app to talk to emulators.
 
+### Developing within a user project
+
+To run the emulator suite in your project folder:
```
Comment (yuchenshi) [inline README.md:29]:
```suggestion
You can also start the dev server of GUI and connect to your real project. To do so, first start the emulator suite in your project folder:
```

File: README.md [hunk 4]
```diff
@@ -1,18 +1,16 @@
 # Firebase Tools UI
 
-A local web UI for Firebase Emulator Suite. Features include:
+The web UI for Firebase Emulator Suite. Features include:
 
 - Overview of Emulators running
 - Firebase Realtime Database Data Viewer/Editor
 - More to come...
```
Comment (yuchenshi) [inline README.md:7]:
```suggestion
- Cloud Firestore Data Viewer/Editor
- Logs Viewer with powerful filters
```

**Comment by tstirrat** [comment]
It should not have force pushed. it should have merged :/

**Pull request merged**